CHANGES.txt

```diff
@@ -21,6 +21,9 @@ Changes
 Fixes
 =====
 
+ - Fixed an issue that caused a table that is not part of the "doc" schema to be
+   unavailable/hidden if it gets closed using ``ALTER TABLE``.
```

Comment:
Can we please be more specific here? The issue was about renaming a closed table that wasn't part of the doc schema right?

Reply:
@andreidan sry, I forgot to update the PR description. I figured out that the root cause was that the table got in a "hidden" state if the table gets closed. So I reflected this in the new changelog entry.